src/App.js

```diff
@@ -1,24 +1,41 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
```

Comment:
- [ ] Please, remove this old piece of commented code, in order to improve the cleanliness of the repo 💪